package.json

```
@@ -14,16 +14,21 @@
 		"name": "Human Made",
 		"url": "https://humanmade.com"
 	},
-	"main": "src/index.js",
+	"main": "./dist/index.js",
+	"source": "./src/index.js",
```

The supplied path will, by default, be treated relative to the current folder, so there is no leading dot necessary (e.g., npm will use `index.js` as default value).

What is `source`? I don't think this is a regular `package.json` property, is it? Unless we need it for anything, can we remove?

```suggestion
	"main": "dist/index.js",
```